.github/workflows/update-browserslist.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Browserslist DB
+on:
+  - workflow_call
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
Why are you using a matrix for one node version and one npm version?

Reply:
we are using matrix in this workflow so that if we want to upgrade the version of npm or node, we can support legacy versions as well as new versions. Arbi-BOM is following this pattern in their workflows as well